src/components/Blogs/index.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../style/variables";
+
+.blogsMainContainer {
+  text-align: center;
+  font-family: $mainFontFamily;
+
+  .container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .col {
+    width: 66.7rem;
+    margin-left: 1.5%;
+  }
+
+  .mr-sm-2 {
+    width: 100%;
+  }
+
+  .blogImgs {
+    width: 21.875rem;
+    height: 21.875rem;
+    padding: 4%;
+    border-radius: 11%;
+  }
+
+  .blogTitles {
+    text-align: center;
+    position: absolute;
```

Comment:
i wonder if this position absolute will create any issues in future? i had to delete from seekHelp section in homepage because it was rendering not the way we wanted

Reply:
absolute position should generally be avoided where possible because it makes code hard to change

Reply:
I used it inside nesting to avoid any kind of conflict in the future. tried different things, absolute was the most accurate one to use. I'll search if there is something else I can use

Reply:
my concern is that it will create a lot of problems with responsiveness. but it's your call in any case.